### ISSUE TITLE
Tichu (1421390)

### ISSUE THREAD

**Issue by pwinckles** [ValveSoftware/Proton#6679]
# Compatibility Report
- Name of the game with compatibility issues: Tichu
- Steam AppID of the game: 1421390

## System Information
- GPU: AMD integrated GPU (Cezanne [Radeon Vega Series / Radeon Vega Mobile Series])
- Driver/LLVM version: Mesa 22.1.7
- Kernel version: 6.2.8
- Link to full system information report as [Gist](https://gist.github.com/pwinckles/eceb0b2d97dcf7b219e67c58691788e9):
- Proton version: Experimental

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

[steam-1421390.log](https://github.com/ValveSoftware/Proton/files/11216617/steam-1421390.log)

## Symptoms

The game crashes while launching.

## Reproduction

Launch the game, and see that it exits without starting.

**Comment by kisak-valve** [comment]
Hello @pwinckles, `Unhandled exception: unimplemented function Cards.dll.??0TClientId@lewarcards@@QAE@ABV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@W4TSeat@1@@Z called in 32-bit code (0x7bc2f6b0).` looks like the line of interest from your log.

**Comment by pwinckles** [comment]
Looking at the log, I noticed that all of the application dlls were loaded as native, except for Cards.dll, which was loaded as builtin -- I have no idea why. I tried adding `WINEDLLOVERRIDES="Z:\\home\\pwinckles\\.local\\share\\Steam\\steamapps\\common\\Tichu\\Cards.dll=n"` to my launch opts, and the game does now start. However, it is still not playable. Here are the issues I see now:

1. Clicking "create table" results in an error message that says "cannot connect socket (host=localhost, port=7319)"
2. It is impossible to close the game without a force-quit.
